docs/user/manual.adoc

```diff
@@ -335,13 +382,44 @@ If the LSP binary is not available, GNOME Builder can install it when opening a
 
 **Source:** https://github.com/rust-analyzer/rust-analyzer/blob/master/crates/rust-analyzer/src/config.rs[config.rs]
 
-rust-analyzer is configured via LSP messages, which means that it's up to the editor to decide on the exact format and location of configuration files.
-Please consult your editor's documentation to learn how to configure LSP servers.
+The <<_installation,Installation>> section contains details on configuration for some of the editors.  In
+general `rust-analyzer` is configured via LSP messages, which means that it's up to the editor to decide on
+the exact format and location of configuration files.
+
+Some clients, such as <<vs-code,VS Code>> or <<coc-rust-analyzer,COC plugin in Vim>> provide `rust-analyzer`
+specific configuration UIs. Others may require you to know a bit more about the interaction with
+`rust-analyzer`.
+
```

Comment:
```suggestion
Some clients, such as <<vs-code,VS Code>> or <<coc-rust-analyzer,COC plugin in Vim>> provide `rust-analyzer` specific configuration UIs. 
Others may require you to know a bit more about the interaction with`rust-analyzer`.
```

We use easier to diff sentence-per-line formt for long docs:

https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md#documentation

Reply:
Got it.  Fixed both commits to use the sentence-per-line format.
I was looking at line 13 and assumed that some lines are wrapped on the 110 character boundary.
Thanks for the link.